nb-tests/test_notebooks.py

```diff
@@ -0,0 +1,57 @@
+import os
+import pathlib
+from dataclasses import dataclass
+import papermill
+from papermill.execute import PapermillExecutionError
+import pytest
+
+
+NB_DIR = pathlib.Path(__file__).parent.parent / "doc" / "source" / "notebooks"
+
+KNOWN_ERRORS_REGEXES = {  # This is basically a GitHub ticket queue
+    "EarthRanger_IO.ipynb": "Series found",
+    "GEE_IO.ipynb": "frontend does not support input requests", # This error only happens in the ci pipeline
+    "EcoMap.ipynb": "frontend does not support input requests", # This error only happens in the ci pipeline
+    "Remote Sensing Time Series Anomaly.ipynb": "frontend does not support input requests", # This error only happens in the ci pipeline
```

Comment:
Do you think we can resolve the errors marked as

```python
"frontend does not support input requests"  # This error only happens in the ci pipeline
```
before merging?

IMHO, it makes sense to not fix _all_ "KNOWN ERRORS" before merging, since many of these are not problems specific to the CI pipeline. But if the error itself if specific to the CI pipeline, I think it makes sense to fix that here. One reason being is that the notebooks failing in this way may have other "actual" errors in them which are not being caught because this error is being thrown first.

If I had to guess, these errors are being thrown because we are not providing the correct environment variables to the Earth Engine initialization? And therefore we are hitting this block

https://github.com/wildlife-dynamics/ecoscope/blob/3caa5e6b754e119af263f1ee0ec7dd706db83ae9/ecoscope/contrib/geemap.py#L127-L129

which presumably is asking for user input to move forward?

Optimistically, I'd think the fix is "just" a matter of translating the GEE credentials we are using in the workflow into the correct format to be picked up here:

https://github.com/wildlife-dynamics/ecoscope/blob/3caa5e6b754e119af263f1ee0ec7dd706db83ae9/doc/source/notebooks/01.%20IO/GEE_IO.ipynb#L91-L92

?

I would assume `EE_PRIVATE_KEY_DATA` is _probably_ the `json.load`'ed representation of `GEE_SERVICE_ACCOUNT_KEY`. And the `EE_ACCOUNT` is the username associated with that account?

There is a possibility once these variables are parsed correctly, that we will find that the secret configured here is not in fact registered with GEE. (That is, I'd guess once "frontend does not support input requests" is fixed, we will see something like "This account does not exist on GEE".) In which case we will need to resolve that as well.